src/utils/helpers/fetchMethods.ts

```diff
@@ -27,8 +27,10 @@ export async function postRequest({
       },
       body,
     })
-    return response.json()
+    const responseJSON = await response.json()
+    return responseJSON
```

Comment:
No actual change here. I wanted to inject a general handler here to notify and logout for any failed request, but it was getting too complicated for right now.

Reply:
Did you want to leave the console logs here?

Reply:
Yeah, cos it's displaying an error for a state that shouldn't really occur. So if it does, it'd be good to see a console error. As I said elsewhere, I'd like to put some proper, generic error handling into these methods, but will make do for now.